### PR TITLE
Minor fixes on hpc setup guide for cheyenne

### PR DESCRIPTION
**Pull request by leosiqueira**
updating conda command usage for newer version where ``source activate`` was replaced with ``conda activate`` and added account name for interactive qsub.

### CODE REVIEW

File: docs/setup_guides/hpc.rst [hunk 1]
```diff
@@ -79,7 +97,8 @@ this section.)
 .. note::
 
    When using recent Jupyter iteration the following section can be replaced by simply invoking the command::
```
Comment (guillaumeeb) [inline docs/setup_guides/hpc.rst]: So we should make commands wit plural?

Reading the whole documentation, I think we don't want to ignore the section completly, as the two last steps should be done either case. Maybe we should add a subsection for setting the password?

File: docs/setup_guides/hpc.rst [hunk 2]
```diff
@@ -170,15 +189,15 @@ In our case, the Cheyenne super computer uses the PBS job scheduler, so typing:
 
 ::
 
-    (pangeo) $ qsub -I -l select=1:ncpus=4 -l walltime=03:00:00 -q regular
+    (pangeo) $ qsub -I -A account -l select=1:ncpus=4 -l walltime=03:00:00 -q regular
```
Comment (guillaumeeb) [inline docs/setup_guides/hpc.rst]: -A account is required for Cheyenne, but not necessarily with other systems. This is not mandatory in CNES for example.

File: docs/setup_guides/hpc.rst (hunk 1 above)
Comment (leosiqueira) [reply docs/setup_guides/hpc.rst]: Agreed, and the two last steps are indeed important in any case.

File: docs/setup_guides/hpc.rst (hunk 2 above)
Comment (leosiqueira) [reply docs/setup_guides/hpc.rst]: It is indeed not required in many other systems, but at the beginning the page says the examples were developed using NCAR's Cheyenne and the sentence just before this says "In our case, the Cheyenne ... uses the PBS job scheduler, so typing:".

Comment (guillaumeeb) [reply docs/setup_guides/hpc.rst]: OK, fine by me.

**Comment by rabernat** [comment]
Is this ready to merge?

**Comment by leosiqueira** [comment]
I think so for now. Guillaume is suggesting that we also should consider changing the config section layout at some point.

**Comment by guillaumeeb** [comment]
So let's merge this, we'll improve the rest later on.

**Pull request merged**